12/umbraco-cms/content-delivery-api.md

```diff
@@ -0,0 +1,470 @@
+
+```json
+{
+  "name": "string",
+  "route": {
+    "path": "string",
+    "startItem": {
+      "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "path": "string"
+    }
+  },
+  "id": "11fb598b-5c51-4d1a-8f2e-0c7594361d15",
+  "contentType": "string",
+  "properties": {
+    "property1Alias": "string",
+    "property2Alias": 0,
+    "property3Alias": true,
+    "property4Alias": [],
+    "property5Alias": {},
+    "property6Alias": null
+  },
+  "cultures": {
+    "cultureIsoCode1": {
+      "path": "string",
+      "startItem": {
+        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "path": "string"
+      }
+    },
+    "cultureIsoCode2": {
+      "path": "string",
+      "startItem": {
+        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "path": "string"
+      }
+    }
+  }
+}
+```
+
+</details>
+
+<details>
+
+<summary>Start item</summary>
+
+When working headlessly in a multi-site setup, sometimes it can be difficult to determine the site context of a particular content item. This is where the _start item_ comes into play.
+
+The start item represents the root of a content item and is commonly returned from the API in conjunction with a content path. For instance:
+
+```json
+{
+  "route": {
+    "path": "/articles/2023/getting-started",
+    "startItem": {
+      "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "path": "docs-portal"
+    }
+  }
+}
+```
+
+This means that the content item resides under the root node with "docs-portal" as an alias and can be retrieved using the path "/articles/2023/getting-started".
+
+
+
+The start item is also quite helpful when obtaining content from the API. By supplying either the start item `id` or `path` in the `Start-Item` header, you can instruct the Delivery API to use the corresponding root node as the starting point for the requested content operation:
+
+```http
+GET /umbraco/delivery/api/v1/content/item/articles/2023/getting-started
+Start-Item: docs-portal
+```
+
+</details>
+
+<details>
+
+<summary>Output expansion</summary>
+
+**Output expansion** allows you to retrieve additional data about related content in the API output for a given content item.
+
+By default, a content property that allows picking a different content item (for example a content picker property) outputs a shallow representation of the picked item. That means, only the basic information about the picked item, without the item properties. However, with output expansion, it is possible to include the properties of the picked item in the API output.
+
+This feature can be used both when querying for single and multiple content items, by adding an `expand` parameter to the query. The value of this parameter can be either `"all"` to expand all properties of the requested content item or `"property:alias, alias, alias"` to expand specific ones.
+
+
+
+The following JSON snippet demonstrates the default output of a content item (without expanding any properties):
```

Comment:
There are two snippets, right? 🤔 

```suggestion
The following JSON snippets demonstrate the default output of a content item (without expanding any properties).
```

Reply:
Only the one right after shows "_default output of a content item (**without** expanding any properties)_". The second one will expand the properties. So "snippet" should be singular here 🙂